package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "6.13.0",
+  "version": "6.14.0-rc.8",
```

Comment:
Why 8? 😅 Were you testing the new action?

Reply:
Yeah, not found a way to be able to test locally first. That's why it got bumped until 8. 😅